USAGE_DATA.md

```diff
@@ -24,4 +24,6 @@ The implementation of telemetry collection can be found in https://github.com/re
 Red Hat extensions' specific telemetry collection details can be found there:
 
 * [Dependency Analytics](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/blob/master/Telemetry.md)
-* [OpenShift Connector](https://github.com/redhat-developer/vscode-openshift-tools/blob/master/USAGE_DATA.md)
\ No newline at end of file
+* [OpenShift Connector](https://github.com/redhat-developer/vscode-openshift-tools/blob/master/USAGE_DATA.md)
+* [Project Initializer](https://github.com/redhat-developer/vscode-project-initializer/blob/master/USAGE_DATA.md)
+* [Remote Server Protocol](https://github.com/redhat-developer/vscode-rsp-ui/blob/master/USAGE_DATA.md)
```

Comment:
shouldn't it be "Remote Server Protocol UI"?

Reply:
which is a horrible name BTW ;-)

Reply:
The UI is a technical qualifier due to the fact that we splited the extension into several. SO I thought talking about the general concept would be more user friendly

Reply:
Ok I can live with that.